com.unity.sg2/Editor/GraphDeltaRegistry/FunctionDefinitions/StandardDefinitions/Input/2D/LightTextureNode.cs

```diff
@@ -0,0 +1,74 @@
+using Usage = UnityEditor.ShaderGraph.GraphDelta.GraphType.Usage;
+
+namespace UnityEditor.ShaderGraph.Defs
+{
+    internal class LightTextureNode : IStandardNode
+    {
+        public static string Name => "LightTexture";
+        public static int Version => 1;
+
+        public static NodeDescriptor NodeDescriptor => new(
+            Version,
+            Name,
+            functions: new FunctionDescriptor[] {
+                new (
+                    "LightTex0",
+                    "Out = UnityBuildTexture2DStructNoScale(_ShapeLightTexture0);",
+                    new ParameterDescriptor[]
+                    {//TODO: need a light texture type, or a light tetexture REF
+                        new ParameterDescriptor("Out", TYPE.Texture2D, Usage.Out)
+                    }
+                ),
+                new (
+                    "LightTex1",
+                    "Out = UnityBuildTexture2DStructNoScale(_ShapeLightTexture1);",
+                    new ParameterDescriptor[]
+                    {
+                        new ParameterDescriptor("Out", TYPE.Texture2D, Usage.Out)
+                    }
+                ),
+                new (
+                    "LightTex2",
+                    "Out = UnityBuildTexture2DStructNoScale(_ShapeLightTexture2);",
+                    new ParameterDescriptor[]
+                    {
+                        new ParameterDescriptor("Out", TYPE.Texture2D, Usage.Out)
+                    }
+                ),
+                new (
+                    "LightTex3",
+                    "Out = UnityBuildTexture2DStructNoScale(_ShapeLightTexture3);",
+                    new ParameterDescriptor[]
+                    {
+                        new ParameterDescriptor("Out", TYPE.Texture2D, Usage.Out)
+                    }
+                ),
+            }
+        );
+
+        public static NodeUIDescriptor NodeUIDescriptor => new(
+            Version,
+            Name,
+            displayName: "2D Light Texture",
+            tooltip: "A 2D Light texture asset.",
+            category: "Utility/Input/2D",
```

Comment:
IMO, this should go in the Input/Lighting category instead of in this weird category all by itself.